sdk/eventgrid/azure-eventgrid/azure/eventgrid/_models.py

```diff
@@ -131,19 +134,24 @@ def _to_generated(self, **kwargs):
 
 class EventGridEvent(InternalEventGridEvent, EventMixin):
     """Properties of an event published to an Event Grid topic using the EventGrid Schema.
+    Note: EventGridEvents can only be sent to a topic of EventGridEvent schema.
+    For CloudEvent schema, use CloudEvent.
 
     Variables are only populated by the server, and will be ignored when sending a request.
 
     All required parameters must be populated in order to send to Azure.
 
     :param topic: The resource path of the event source. If not provided, Event Grid will stamp onto the event.
+        Is required for Domain topics, and a domain topic may be specified which doesn't yet exist.
```

Comment:
also, it seems over indented, not sure it would cause rendering issue
```suggestion
     Is required for Domain topics, and a domain topic may be specified which doesn't yet exist.
```